src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -104,24 +104,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Before destroying this instance we must ensure that we destroy the _renderer
         // before the _renderEngine, as well as calling _renderer->TriggerTeardown().
         // _renderEngine will be destroyed naturally after this ~destructor() returns.
-
-        decltype(_renderer) renderer;
-        {
-            // GH#8734:
-            // We lock the terminal here to make sure it isn't still being
-            // used in the connection thread before we destroy the renderer.
-            // However, we must unlock it again prior to triggering the
-            // teardown, to avoid the render thread being deadlocked. The
-            // renderer may be waiting to acquire the terminal lock, while
-            // we're waiting for the renderer to finish.
-            auto lock = _terminal->LockForWriting();

```

Comment:
(we don't need to lock any longer?)

Reply:
The core thesis of the fix earlier was that we needed to lock to steal away the renderer. I know the rest of the fix pulled the code out of Close so that _renderer continued to exist ... so perhaps this is not a problem

Reply:
This line was only added because j4james (not tagging to not spam them) was experiencing crashes. But as we now know we can fix it much more thoroughly by ensuring that no one is using the renderer anymore.
Originally there was no `LockForWriting` before calling `TriggerTeardown`. Assuming this was correct back then, we won't need it anymore now.